benchmark/inclusion/run.sh

```diff
@@ -1,17 +1,20 @@
 #!/bin/bash
 
 # global refinement:
-for r in "3" "4" "5" "6" "7" "8"
+echo "--Global Refinement--"
+for r in "4" "5" "6" "7" "8" "9"
 do
 echo "ref $r:"
 cp global.prm.base temp.prm
-echo "subsection Mesh refinement" >>temp.prm
+echo "set Output directory = output/global/ref$r" >> temp.prm
+echo "subsection Mesh refinement" >> temp.prm
```

Comment:
If you really want to run your model for different averaging algorithms (as your description of the pull request implies), this is the place where it should go. 
You can do another loop (over whatever averaging operations you want to include here)

for avg in "none" "arithmetic average" "geometric average" "harmonic average" "pick largest" "project to Q1"
do
        ... (the stuff you already have)
	echo "subsection Material model" >>temp.prm
	echo "set Material averaging = $avg" >> temp.prm
	echo "end" >> temp.prm

And then this will be changed automatically in the input file. And then you don't even need the line 'set Material averaging = none' in the input file you added above.